### PR TITLE
feat: add responsive templates

### PR DESCRIPTION
**Pull request by JasonLantz**
templates minus the goofy sidebar break option......

### CODE REVIEW

File: docs/elements/grid.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+You should generally not use this component directly, prefer instead to use Templates and Presentation Strategies in your designs.
+
+## Grid Structure
+The grid consists of four key parts: the Container, Container Margins, Columns, and Gutters.
+
+All columns reside within the Container, which is located at the root level of a page. Depending on the breakpoint, the container will have Container Margins of various sizes. The container also has a maximum width property, beyond which only the container margins grow in size.
+
+The container is divided into 12 proportional Columns which contain actual content. These are separated by Gutters of a fixed size (which varies based on the breakpoint again).
+
+See the redlines for the exact dimensions of each feature at each breakpoint.
```
Comment (StommePoes) [inline docs/elements/grid.md]: Kunnen wij hier ook een link toevoegen naar de Design pagina redlines?

File: docs/elements/templates.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+{{/demo}}
+
+
+## Shared Features
+Spacing:
+- Directly adjoining templates have no vertical space between them
+- Small Gap is 40px total
+- Large Gap is 80px total
+
+Backgrounds & Borders:
```
Comment (StommePoes) [inline docs/elements/templates.md]: and amp ;

**Comment by StommePoes** [comment]
If "some_rule :last-child" (with the space) can't accidentally hit descendents, this looks good.

File: docs/elements/grid.md (hunk 1 above)
Comment (JasonLantz) [inline docs/elements/grid.md]: We konden.... Maar mijn docs zou droevige bilk......

Comment (JasonLantz) [inline docs/elements/grid.md]: Or just add a link and leave the text....

Comment (StommePoes) [inline docs/elements/grid.md]: That's what I thought, add the link to the text you've got.
I dunno bilk, that's English for stealing money right? :P

Comment (JasonLantz) [inline docs/elements/grid.md]: Good question... We should ask google.... :)
I was saying that my docs would look sad if I pulled the text...

Cool, I added the link to the red lines and pushed out a new pr...

Comment (StommePoes) [inline docs/elements/grid.md]: Yeah yeah I wasn't suggesting losing the text! It needs to be in here. It's just on the Design site they can also admire the pwetty pictures too :)

**Pull request closed without merging**